ioc_rix_sp1k1_calc/__main__.py

```diff
@@ -1,17 +1,35 @@
 import textwrap
+from os import environ
+from pathlib import Path
+from shutil import copy2
 
 import caproto.server
 
 from ioc_rix_sp1k1_calc import Ioc_rix_sp1k1_calc
 
 
+def install_archive(pv_names):
+    ioc_name = "ioc-rix-sp1k1-calc"
+    archive_file = ioc_name + '.archive'
+    data_dir = environ['IOC_DATA']
+    if not data_dir:
+        data_dir = '/reg/d/iocData'
```

Comment:
I'm not sure when the /reg dir is going away, but it's definitely deprecated:
```suggestion
        data_dir = '/cds/data/iocData'
```

Reply:
I need to go update the filepaths in all the epics startup scripts...